### PR TITLE
Document configurations for serving-only compute nodes and node-specific parameters

### LINKED ISSUE #1510: Document configurations for serving-only compute nodes
https://github.com/risingwavelabs/risingwave/pull/13239

It is related to batch serving mode.
It is also related to performance tuning. 

Guess it can be referred to in both places.
I try to make the config file itself as self-contained as possible.
If I miss anything or express unclearly in the `toml` file, please let me know and I will fix that in the kernel repository too.

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info

- **Description**

  - Add "[Configure node-specific parameters](https://pr-1687.d2fbku9n2b6wde.amplifyapp.com/docs/dev/configure-node-specific-parameters/)."
  - Update "Dedicated compute node" to include "[Configuring a serving compute node for batch queries](https://pr-1687.d2fbku9n2b6wde.amplifyapp.com/docs/dev/dedicated-compute-node/#configuring-a-serving-compute-node-for-batch-queries)."

- **Notes**

  - Need inputs for "[Node-specific parameters](https://pr-1687.d2fbku9n2b6wde.amplifyapp.com/docs/dev/node-specific-parameters/)."
  - @MrCroxx Please check and edit directly the steps in "[Setting up node-specific parameters](https://pr-1687.d2fbku9n2b6wde.amplifyapp.com/docs/dev/configure-node-specific-parameters/#setting-up-node-specific-parameters)."

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/issues/13149

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1510

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Comment by hengm3467** [comment]
@lmatz @neverchanje @Little-Wallace @MrCroxx  Please edit the node-specific parameters in the [Node-specific parameters](https://pr-1687.d2fbku9n2b6wde.amplifyapp.com/docs/dev/node-specific-parameters/) topic. 

This PR also resolves #1630 .

**Review by lmatz — APPROVED** [review]
The `docs/manage/dedicated-compute-node.md` part LGTM

File: docs/manage/configure-node-specific-parameters.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+reclaimers = 4
+recover_concurrency = 8
+lfu_window_to_cache_size_ratio = 1
+lfu_tiny_lru_capacity_ratio = 0.01
+insert_rate_limit_mb = 0
+reclaim_rate_limit_mb = 0
+ring_buffer_capacity_mb = 256
+catalog_bits = 6
+compression = "none"
+```
```
Comment (MrCroxx) [inline docs/manage/configure-node-specific-parameters.md]: Block `[storage.cache_refill]` should also be added here.

Comment (MrCroxx) [reply docs/manage/configure-node-specific-parameters.md]: BTW, we should add a standalone documentation to explain how to configure the file cache and refill.

**Review by MrCroxx — COMMENTED** [review]
Rest LGTM.

File: docs/manage/configure-node-specific-parameters.md (hunk 1 above)
Comment (hengm3467) [reply docs/manage/configure-node-specific-parameters.md]: @MrCroxx Could you please draft a summary of the steps to do that? Thanks.

**Comment by hengm3467** [comment]
@neverchanje @lmatz @Little-Wallace Do we have a list of node-specific parameters? Thanks.

**Comment by WanYixian** [comment]
> @neverchanje @lmatz @Little-Wallace Do we have a list of node-specific parameters? Thanks.

This is done now, see [Node-specific parameters](https://pr-1687.d2fbku9n2b6wde.amplifyapp.com/docs/dev/node-specific-parameters/) for preview. Will merge this PR if no further questions. Thanks everybody!

**Comment by lmatz** [comment]
> > @neverchanje @lmatz @Little-Wallace Do we have a list of node-specific parameters? Thanks.
> 
> This is done now, see [Node-specific parameters](https://pr-1687.d2fbku9n2b6wde.amplifyapp.com/docs/dev/node-specific-parameters/) for preview. Will merge this PR if no further questions. Thanks everybody!

`node-specific-parameters` page seems related to storage only, I will defer to @MrCroxx @Little-Wallace @Li0k  to decide.

**Comment by MrCroxx** [comment]
I'm okay with the `node-specific-parameters` part. 🥰

**Pull request merged**